lib/msf/core/auxiliary/scanner.rb

```diff
@@ -25,6 +25,7 @@ def initialize(info = {})
   deregister_options('RHOST')
 
   register_advanced_options([
+    OptBool.new('RandomizeHostOrder', [true, 'Randomize the order in which hosts are scanned', true]),
```

Comment:
I think this should be `false` by default unless we are OK with the default behavior now being different.